### PR TITLE
added fix to docker image

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/34667

## Description
- In case of creating a new integration in an existing pack (e.g, as listed in the issue above), the docker image RN was added although it shouldn't, as the integration was created for the first time, and only a RN describing the new integration should be added.

Fixed it by checking commit vs master,  if the YML file existed in the first place before adding the docker image update to the RN

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ShahafBenYakir — COMMENTED** [review]
I think we can save this operation. The `execute_update` function (which calls this one you changed) already have this data, so we can do something like this:
```
is_integration = if 'yml' in packfile and file_type == FileType.INTEGRATION:
is_new_file = packfile in self.added_files
if is_integration and not is_new_file:
    is_docker_image_changed, docker_image_name = check_docker_image_changed(packfile)
changed_files[file_name] = {
    'type': file_type,
    'description': get_file_description(packfile, file_type),
    'is_new_file': is_new_file
    'fromversion': get_from_version_at_update_rn(packfile)
}
```

**Comment by ShahafBenYakir** [comment]
- Need to add a unit test
- Need to add this RN to the changelog

**Comment by tomneeman151293** [comment]
@ShahafBenYakir Done!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #Changelog
+* Fixed an issue where **update-release-notes** on newly created integration added release notes with '- Updated the Docker image' note.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:2]:
```suggestion
* Fixed an issue where **update-release-notes** on newly created integration added release notes for docker update.
```

File: demisto_sdk/commands/update_release_notes/tests/update_rn_test.py [hunk 2]
```diff
@@ -1128,6 +1128,42 @@ def test_add_and_modify_files_without_update_docker_image(self, mocker):
         os.remove('demisto_sdk/commands/update_release_notes/tests_data/Packs/release_notes/1_1_0.md')
         assert 'Updated the Docker image to: *dockerimage:python/test:1243*' not in RN
 
+    def test_new_integration_docker_not_updated(self, mocker):
+        """
+        Given
+            - New integration created.
```
Comment (ShahafBenYakir) [inline demisto_sdk/commands/update_release_notes/tests/update_rn_test.py:1134]: Also pack exist right?

**Review by ShahafBenYakir — COMMENTED** [review]
Looks good. Please attach a link to a build containing the failed unit-test (without your fix) so I can see it catches this.

File: demisto_sdk/commands/update_release_notes/tests/update_rn_test.py (hunk 2 above)
Comment (tomneeman151293) [reply demisto_sdk/commands/update_release_notes/tests/update_rn_test.py:1134]: doesn't really matter if pack existed or not

**Comment by tomneeman151293** [comment]
@ShahafBenYakir 
link to build failure
https://app.circleci.com/pipelines/github/demisto/demisto-sdk/8628/workflows/461a60af-735b-4bbd-b081-e1fab276fb7e/jobs/35601

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Changelog
 * Fixed an issue where **lint** failed where *.Dockerfile* exists prior running the lint command.
 * Added FeedHelloWorld template option for *--template* flag in **demisto-sdk init** command.
-
+* Fixed an issue where **update-release-notes** on newly created integration added release notes for docker update.
```
Comment (ShahafBenYakir) [inline CHANGELOG.md:4]:
```suggestion
* Fixed an issue where running **update-release-notes** on a pack with newly created integration, had also added a docker image entry in the release notes.
```

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**